Userland/Libraries/LibWeb/Layout/FormattingContext.h

```diff
@@ -107,6 +107,12 @@ class FormattingContext {
         CSSPixels right { 0 };
     };
 
+    struct SpaceUsedAndEdgeForFloats {
+        SpaceUsedByFloats space;
+        CSSPixels left_end_x { 0 };
+        CSSPixels right_end_x { 0 };
```

Comment:
What do these values represent? Can we give them names that better describe what they contain?

Reply:
They're the space plus the offset from the containing block chain margins from root. I'll think about a more suggestive name, it's a bit tricky to capture it in a short name.